docs/admin/upgrading.rst

```diff
@@ -89,205 +81,76 @@ And replace it with:
-
-PHP 5.5 has reached end of life in July 2016. To ensure that Elgg sites are secure, we now require PHP 5.6 for new installations.
-
-Existing installations can continue using PHP 5.5 until Elgg 3.0.
-
-In order to upgrade Elgg to 2.3 using composer while using PHP 5.5, you may need to use ``--ignore-platform-reqs`` flag.
-
-Tests
------
-
- * PHPUnit bootstrap is deprecated by composer autoloader: Tests should no longer bootstrap themselves using ``/engine/tests/phpunit/bootstrap.php``. Instead, tests should extend ``\Elgg\TestCase``.
- * PHPUnit bootstrap no longer sets global ``$CONFIG``. Tests should use ``_elgg_config()`` instead.
- * Core and tests no longer use private global values in ``$_ELGG->view_path`` and ``$_ELGG->allowed_ajax_views``
-
-Schema
-------
-
- * The database GUID columns need to be aligned. In the admin section an upgrade is available to handle this. Please make sure you have a backup available
-
-From 2.3 to 3.0
-===============
-
-Data removal
-------------
-
-Be aware the 3.0 upgrade process will remove any remaining "legacy" password hashes. This will affect users who have never logged in under an Elgg 1.10 or later system. These users will be politely asked to reset their password.
-
-From 1.x to 2.0
-===============
-
-Removed plugins
----------------
-
-The following plugins are no longer bundled with Elgg core:
-
- * categories (https://github.com/elgg/categories)
- * zaudio (https://github.com/elgg/zaudio)
+If you had your Elgg 2.3 project installed using composer, you can follow this sequence:
 
-IE-specific workarounds have been dropped
------------------------------------------
+**Back up your database, data directory, and code**
 
-Several views (``css/ie``, ``css/ie7``, ``css/ie8``, etc.) as well as conditional
-comments have been discarded now that IE10+ browsers are more standards-compliant.
-If you need browser support farther back than that, you will need to find or build
-a plugin that introduces its own compatibility layer or polyfills.
 
-Update your webserver config
-----------------------------
+.. code-block:: sh
 
-URL paths like ``cache/*`` and ``rewrite.php`` now use the main front controller
-script. You **must** remove these rewrite rules from your webserver config (e.g. ``.htaccess``).
+    composer self-update
 
-Also remove the rules for paths like ``export/*``; these endpoints have been removed.
+    cd ./path/to/project/root
+    composer require elgg/elgg:~3.0
+    composer update
+    vendor/bin/elgg-cli upgrade async -v
 
-Settings location
------------------
 
-After upgrading, move your ``settings.php`` file from ``engine/`` to ``elgg-config/``.
+3b. Manual Upgrade (legacy approach)
+------------------------------------
 
-From 1.10 to 1.11
-=================
+Manual upgrades are a major undertaking for site admins. We discourage you from maintaining an Elgg installation using
+ZIP dist packages. Save yourself some time by learning how to you composer and version control (git).
```

Comment:
We should just stop making dists

Reply:
not everybody has command-line access unfortunately

Reply:
That's not our problem. It's too much work both for the team and the people that use the zip without thinking ahead.